hack/update-swagger-spec.sh

```diff
@@ -18,7 +18,7 @@ function cleanup()
     rm -rf "${TEMP_DIR}"
 
     if [ $out -ne 0 ]; then
-        echo "[FAIL] !!!!! Generate Failed !!!!"
+        echo -e "\033[0;31m [FAIL] !!!!! Generate Failed !!!!\033[0m"
```

Comment:
> Borrowed from Kubernetes verification script style.

Still don't like it.  You could add all the logic for making sure you aren't messing with piped output or eliminate it.